components/contributions/ContributionDrawer.tsx

```diff
@@ -401,147 +364,195 @@ export function ContributionDrawer(props: ContributionDrawerProps) {
                             </Link>
                           }
                         />
+                      )
+                    }
+                  />
+                </InfoList>
+
+                <DataList className="mb-4">
+                  <DataListItem>
+                    <DataListItemLabel>
+                      <FormattedMessage defaultMessage="Amount" id="Fields.amount" />
+                    </DataListItemLabel>
+                    <DataListItemValue>
+                      {isLoading ? (
+                        <Skeleton className="h-5 w-32" />
+                      ) : (
+                        <React.Fragment>
+                          <FormattedMoneyAmount
+                            showCurrencyCode={true}
+                            currency={query.data.order.totalAmount.currency}
+                            amount={query.data.order.totalAmount.valueInCents}
+                          />
+                        </React.Fragment>
                       )}
-                    </div>
-                  </div>
-                  <div>
-                    <div>
+                    </DataListItemValue>
```

Comment:
Maybe we could include a similar frequency hint as in the table to the amount field here (or move the Frequency item just below the amount)
<img width="574" alt="image" src="https://github.com/user-attachments/assets/53870be0-068b-45b3-8a5d-6268fddb2190" />